### ISSUE TITLE
storagenode info reports values from config.yaml, instead of actual ones

### ISSUE THREAD

**Issue by toconspiracy** [storj/storj#6784]
**Description**

When i try to get the info of a Node (for connecting it to the Multinodedashbaord) via CLI within a Docker Node, it displays the values within the config.yaml instead of the ones which are used. This does not need to be correct as (in my case) i overwrite those values via the Docker command variable. 

**Steps to reproduce the issue:**

1. start a storagenode via Docker and configure it via command  `--console.address=:<whateverip>:<whateverport>`
2. have a different value within the config.yaml for `contact.external-address: <anotherip>:<anotherport>`
3. execute the info command within the container: `/app/storagenode info --config-dir /app/config/`

**Describe the results you expected:**

```
ID: <nodeid>
API Secret: <apisecret>
Public Address: <whateverip>:<whateverport>
```

**Describe the results you received:**
```
ID: <nodeid>
API Secret: <apisecret>
Public Address: <anotherip>:<anotherport>
```

**Comment by AlexeyALeonov** [comment]
Seems you confused `--console.address` for the local dashboard with the `--contact.external-address` option, which is actually a public (advertised) node's address and port (dashboard's port is not used in the MND too).
And this option is working.

**Comment by toconspiracy** [comment]
Yeah you are right mixed up those Parameters. But shouldn't report `Public Address` either `contact.external-address:` or (if defined) the environment variable `ADDRESS`?
